src/asymmetric_cipher.c

```diff
@@ -117,10 +120,9 @@ static int p11prov_rsaenc_encrypt_init(void *ctx, void *provkey,
     return p11prov_rsaenc_set_ctx_params(ctx, params);
 }
 
-static int p11prov_rsaenc_encrypt(void *ctx,
-                                  unsigned char *out, size_t *outlen,
-                                  size_t outsize,
-                                  const unsigned char *in, size_t inlen)
+static int p11prov_rsaenc_encrypt(void *ctx, unsigned char *out, size_t *outlen,
+                                  size_t outsize, const unsigned char *in,
+                                  size_t inlen)
```

Comment:
so this one is doing what I'd like to happen for argument passed to a function as well, see above.